pages/in/widget/iscn/index.vue

```diff
@@ -297,8 +305,27 @@ export default {
       if (!window) return null;
       return window.opener;
     },
-    ipfsURL() {
-      return `https://ipfs.io/ipfs/${this.fingerprint}`;
+    fingerprintURLs() {
+      const fingerprintList = [];
+      for (let i = 0; i < this.fingerprint.length; i += 1) {
+        const fingerprintParts = this.fingerprint[i].split('://');
+        let url = '';
+        switch (fingerprintParts[0]) {
+          case 'ipfs':
+            url = `https://ipfs.io/ipfs/${fingerprintParts[1]}`;
+            break;
```

Comment:
We need to backward support old plugin which directly send ipfs hash into `fingerprint` without `ipfs://`
Can we check if a fingprint starts with `Qm` and has a certain length, if yes assume it is ipfs?

Reply:
Yes, as discussed, also add ipfsHash length === 46 as guard